dat/dat.go

```diff
@@ -103,16 +141,57 @@ func (db *DB) FindByROMName(romPath string, romName string, crc uint32, games ch
 				// If the checksums match
 				for _, ROM := range game.ROMs {
 					if romName == ROM.Name {
+						fmt.Printf("Exact match: %s -> %s\n", romName, ROM.Name)
 						game.Path = romPath
 						game.System = system
 						games <- game
+						game_found.mu.Lock()
+						game_found.found = true
+						game_found.mu.Unlock()
+					} else {
+						var gameName = strings.Split(romName, ".")[0]
+						var gameExt  = strings.Split(romName, ".")[1]
```

Comment:
File may contain multiple dots or even no dots.
I guess you could use `filepath.Ext`.

Reply:
Good call!